reconcile/openshift_resources.py

```diff
@@ -292,6 +300,14 @@ def fetch_provider_route(path, tls_path, tls_version):
         logging.info(msg)
         _log_lock.release()
 
+    host = openshift_resource.body['spec'].get('host')
+    certificate = openshift_resource.body['spec']['tls'].get('certificate')
+    if host and certificate:
+        match = openssl.certificate_matches_host(certificate, host)
```

Comment:
Will this work for wildcards?

Reply:
it will, thanks to:
https://github.com/app-sre/qontract-reconcile/pull/190/files#diff-b1b248c88c2f9d9e7923df0911315418R6